btc_address_dump/wif_util.py

```diff
@@ -51,6 +75,16 @@ def decode_wif(wif: str, compressed_wif: bool = False) -> bytes:
     return private_key
 
 
+def grs_decode_wif(wif: str, compressed_wif: bool = False) -> bytes:
```

Comment:
It seems that grs_decode_wif is the same as decode_wif , can we just use decode_wif?